### DISCUSSION TITLE
Rust-analyzer stops working after a bit

### DISCUSSION THREAD

**Discussion by eatenpancreas** [zed-industries/zed#17824] [Feature Requests]
Hello, whenever i'm trying to work on rust code within the zed editor after a little bit the rust-analyzer stops working and code-completion stops working as well. Is there a way to reset rust-analyzer other than to stop and restart the entirety of the ide each time? It would be handy. Wasn't sure if this was an issue with zed or not so i'm just gonna post it here.

![image](https://github.com/user-attachments/assets/6c8a6998-bf6f-44a7-a277-08fba820d2ed)

**Comment by bajrangCoder** [comment]
Share your zed log and device info to debug it.

Btw there is command in command palette "restart language servers" to restart language server without restarting zed

**Reply by eatenpancreas** [reply]
okay perfect the command palette command works, also how do i find/share zed log?

**Reply by bajrangCoder** [reply]
use `zed log` command from command palette
